.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -30,7 +30,7 @@ This role should rotate between LTS releases
   - [ ] Modify the `PACKAGING_GIT_BRANCH` value in the packaging script (`Jenkinsfile.d/core/package`) to match the release.
   - For more info, refer to [stable](https://github.com/jenkins-infra/release#stable).
 
-- [ ] Create or update release branch in [jenkinsci/packaging](https://github.com/jenkinsci/packaging), e.g. `stable-2.387`.
+- [ ] Create or update release branch in [jenkinsci/packaging](https://github.com/jenkinsci/packaging), e.g. `stable-2.387`. Strike out for new point release.
```

Comment:
Unrelated to this PR, but for point releases, you don't need to modify anything on the packaging branch.